var/spack/repos/builtin/packages/intel-oneapi-compilers-classic/package.py

```diff
@@ -37,6 +37,7 @@ class IntelOneapiCompilersClassic(Package):
         "2021.9.0": "2023.1.0",
         "2021.10.0": "2023.2.0",
         "2021.11.0": "2024.0.0",
+        "2021.11.1": "2024.0.0",
```

Comment:
This is the version for ifort. I don't see a 2021.11.1, just 2021.11.0

Reply:
My bad, we were getting data from oneccl version here...